src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,46 @@
+import classNames from 'classnames';
+import { FC } from 'react';
+import { FilterType } from '../../types/FilterType';
+import { Todo } from '../TodoItem/Todo';
+
+interface Props {
+  todos: Todo[],
+  filterType: FilterType,
+  onChangeFilterType: (filterType: FilterType) => void,
+}
+
+export const Footer: FC<Props> = (props) => {
+  const { todos, filterType, onChangeFilterType } = props;
+
+  const activeTodos = todos.filter(todo => !todo.completed);
+  const completedTodos = todos.filter(todo => todo.completed);
```

Comment:
Is it realy nessesary to filter your todos twise? And without useMemo? I suggest to wrap activeTodos calculation in useMemo. And because you need only size of active and completed todos for completedTodos you can just 'todos.length - activeTodos.length'

Reply:
My final logic for this is in my final implementation) I've used useMemo there)